packages/workbox-build/src/lib/utils/filter-files.js

```diff
@@ -89,7 +89,8 @@ module.exports = (fileDetails, options) => {
 
   if (options.manifestTransforms) {
     if (Array.isArray(options.manifestTransforms)) {
-      manifestTransforms.concat(options.manifestTransforms);
+      manifestTransforms =
+          manifestTransforms.concat(options.manifestTransforms);
```

Comment:
This bug just bit me too, would you folks take a PR that just fixed this? I already was about submit it, when i saw it was covered here. But... basically as it's written right now anything you pass in as a `manifestTransforms` option won't be run at all. Since `.concat` doesn't change an array in place. Effectively rendering this option useless.